.github/workflows/build_docs.yml

```diff
@@ -2,47 +2,41 @@
  on:
    push:
      branches:
-     - master
      - main
+
  jobs:
    docs:
-     name: CI (${{ matrix.os }}-${{ matrix.environment-file }})
+     name: Build & Push Docs
      runs-on: ${{ matrix.os }}
-     continue-on-error: ${{ matrix.experimental }}
-     timeout-minutes: 20
+     timeout-minutes: 90
      strategy:
        matrix:
          os: ['ubuntu-latest']
-         environment-file: [ci/311.yml]
+         environment-file: [ci/311.yaml]
```

Comment:
Should we try for 3.12 here?

Reply:
sure, we can bump one version. There's a bunch of extra crap for the docs in that env